src/components/Dashboard/widgets/Embeds/TransferEmbed.tsx

```diff
@@ -2,30 +2,38 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 
 import styled from '@emotion/styled';
+import isArray from 'lodash-es/isArray';
```

Comment:
I'd suggest against adding more reliance on lodash, if we want to remove it in the future. In this specific use case the native `Array.isArray()` might be just enough.